### DISCUSSION TITLE
Can I add custom text to the changelog and release notes?

### DISCUSSION THREAD

**Discussion by trallnag** [semantic-release/semantic-release#2082] [Q&A] [answered]
From time to time I want to add additional information to changelog and release notes. For example more detailed explanation on changes.

Is this possible with semantic-release?

**Comment by trallnag** [comment]
Partly answered / discussed in https://github.com/semantic-release/semantic-release/discussions/2080

**Comment by travi** [comment, marked as answer]
`@semantic-release/github` will never modify existing release notes in github releases. `@semantic-release/changelog` only appends to the end of your changelog file.

if you have cases where you need to manually modify either of these, you can be confident that semantic-release will not overwrite your changes. are those the cases you are asking about, or are there other scenarios that you are curious about?

**Reply by trallnag** [reply]
>are those the cases you are asking about

Yep. For example adding tips on how to migrate to the new version in case of breaking changes
